packages/client/hmi-client/src/components/model/petrinet/tera-model-intervention.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+	<div class="form">
+		<div class="label-col">
+			<label>Parameter name </label>
+			<Dropdown
+				v-if="parameterOptions"
+				:options="parameterOptions"
+				v-model.lazy="name"
+				@update:model-value="updateIntervention"
+			/>
+			<InputText v-else v-model.lazy="name" @update:model-value="updateIntervention" />
```

Comment:
I don't understand why we have a dropdown and an inputbox, under what usage scenario will parameterOPtions be null?

Reply:
Currently there is no scenario where it is null. I can remove this and make it that required